pages/login.html

```diff
@@ -9,25 +9,29 @@
     <link rel="stylesheet" href="https://component-site-fitui.netlify.app/components/components.css">
     <link rel="stylesheet" href="../css/header.css">
     <link rel="stylesheet" href="../css/login.css">
-    <link rel="stylesheet" href="../style.css">
+    <link rel="stylesheet" href="../css/style.css">
     <script src="https://code.iconify.design/2/2.1.2/iconify.min.js"></script>
 </head>
 
 <body>
     <div class="grid-layout">
         <header class="header">
             <div class="navbar">
-                <h1 class="navbar-title" ><a href="./product.html">Fit Store</a></h1>
+                <h1 class="navbar-title" ><a href="../index.html">Fit Store</a></h1>
```

Comment:
you can also use only / for index.html in href

Reply:
> you can also use only / for index.html in href

so i tried but not working

Reply:
> > you can also use only / for index.html in href
> 
> so i tried but not working

thanks, working now